### PR TITLE
sinokor webscraper works in docker and headless

### PR DESCRIPTION
**Pull request by VaruneshBala**
# Description

BDTT-71 Sinokor Webscraper 
Fixes [Jira ticket #]()

telegram-cloud-photo-size-5-6125111771053274636-y

List all acceptance criterias of user story and indicate which has been solved previously and which is solved currently
- [x] Able to webscrape from Sinokor Website and returns values shown on the SS above (solved)
- [] XXX (solving)
- [ ]

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Refactor
- [ ] Migration

# Checklist:

- [X] I have added the appropriate labels to my PR
- [X] My code follows the style guidelines of this project
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: services/backend/scrapers/sinokorScraper.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        time.sleep(3)
+
+        search_box = driver.find_element(By.XPATH, '/html/body/div[3]/div/div[3]/div[2]/div[2]/div/div[2]/div/input')
+        search_box.send_keys(identifier)
+        time.sleep(3)
+
+        driver.find_element(By.XPATH, '/html/body/div[3]/div/div[3]/div[2]/div[2]/div/div[2]/div/span[2]/button').click()
+        time.sleep(5)
+
+        if identifier_type == "CTR":
```
Comment (pandaponn) [inline services/backend/scrapers/sinokorScraper.py]: all other scrapers are using lower case identifier_type, i.e. "ctr". Should change to lower case.

**Review by pandaponn — APPROVED** [review]
I'm assuming Worldgate didn't provide B/L, so this scraper does not support B/L searches? @VaruneshBala Otherwise, LGTM

File: services/backend/scrapers/sinokorScraper.py (hunk 1 above)
Comment (pandaponn) [reply services/backend/scrapers/sinokorScraper.py]: i've changed this alr

**Comment by VaruneshBala** [comment]
> I'm assuming Worldgate didn't provide B/L, so this scraper does not support B/L searches? @VaruneshBala Otherwise, LGTM

hmm no they provided BL and ctr number. ive tested using both and it works for me

File: services/backend/scrapers/sinokorScraper.py (hunk 1 above)
Comment (VaruneshBala) [reply services/backend/scrapers/sinokorScraper.py]: thank you

**Comment by pandaponn** [comment]
oh yea mb i read the if statement wrongly

**Pull request merged**